public/docs/_examples/dependency-injection/dart/lib/app_component.dart

```diff
@@ -31,21 +32,21 @@ import 'providers_component.dart';
       CarComponent,
       HeroesComponent,
       InjectorComponent,
+      TestComponent,
       ProvidersComponent
     ],
-// #docregion providers
+    // #docregion providers
     providers: const [
-      Logger,
-      UserService,
-      const Provider(AppConfig, useValue: config1)]
-// #enddocregion providers
+      Logger, UserService,
+      const Provider(APP_CONFIG, useFactory: heroDiConfigFactory)]
```

Comment:
The OpaqueToken section will need to be revisited once #1563 gets resolved so I decided to leave the token name like it is in the TS example (because then the token name _and_ the interface name will match, both being `AppConfig`).